### PR TITLE
Update set-up-mdt-for-bitlocker.md

### PR DESCRIPTION
**Pull request by nenonix**
Edits for clarity, adds note format, resolves https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4266

### CODE REVIEW

**Comment by illfated** [comment]
Just a minor thought regarding semantics, I wonder which one is more precise:
> [!NOTE]  It is common for organizations wrapping these tools in scripts to get additional logging and error handling.

versus
> [!NOTE]  It is common for organizations **to wrap** these tools in scripts to get additional logging and error handling.

**Comment by jvsam** [comment]
Since @nenonix created this, for copy review @JohanFreelancer9  :-) Thank you.

File: windows/deployment/deploy-windows-mdt/set-up-mdt-for-bitlocker.md [hunk 1]
```diff
@@ -136,12 +136,14 @@ cscript.exe SetConfig.vbs SecurityChip Active
...
 
-When configuring a task sequence to run any BitLocker tool, either directly or using a custom script, it is helpful if you also add some logic to detect whether the BIOS is already configured on the machine. In this task sequence, we are using a sample script (ZTICheckforTPM.wsf) from the Deployment Guys web page to check the status on the TPM chip. You can download this script from the Deployment Guys Blog post, [Check to see if the TPM is enabled](https://go.microsoft.com/fwlink/p/?LinkId=619549). In the following task sequence, we have added five actions:
+When configuring a task sequence to run any BitLocker tool, either directly or using a custom script, it is helpful if you also add some logic to detect whether the BIOS is already configured on the machine. In the following task sequence, we are using a sample script (ZTICheckforTPM.wsf) from the Deployment Guys web page to check the status on the TPM chip. You can download this script from the Deployment Guys Blog post, [Check to see if the TPM is enabled](https://go.microsoft.com/fwlink/p/?LinkId=619549). 
+
+We added these five actions to the task sequence:
```
Comment (JohanFreelancer9) [inline windows/deployment/deploy-windows-mdt/set-up-mdt-for-bitlocker.md]:
```suggestion
We have added these five actions to the task sequence:
```

**Comment by jvsam** [comment]
Hi @greg-lindsay this pull request has been copy edited and is ready for your final review and approval. Please let us know if there are outstanding issues to be resolved prior to merge. Thank you very much.

**Comment by jvsam** [comment]
Hi again @greg-lindsay I'd like to follow-up on this. We will also appreciate it if you can provide feedback if there are issues with the amendments. Thank you.

**Comment by jvsam** [comment]
Hello again @greg-lindsay hoping you can review this pull request. This PR suggests some minor grammatical improvements to make the instructions less confusing for readers. If this is fine, kindly approve and merge. Thank you.

CC: @AndreaBarr will you be able to review as well please and merge if there are no issues? Thank you.

**Comment by jvsam** [comment]
Thank you for merging this PR @AndreaBarr .

**Pull request merged**